stable/enterprise/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: enterprise
-version: "3.8.0"
+version: "3.8.1"
```

Comment:
We should probably bump this to the next minor version instead of a patch since its changing/adding a new way the chart works. Also we should probably update the README to include the changes (both the kubectlImage change and the image/ui.image change)